<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_coincidencies</class>
 <widget class="QMainWindow" name="w_coincidencies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Multiple coincidencies</string>
     </attribute>
     <widget class="QPushButton" name="ButtonRunAll">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>70</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Experimental">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QPushButton" name="ButtonSelectFolder">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Folder if tracking in several files</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButDelFolder">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFolderPath">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>9</y>
         <width>411</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_Experimental_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>741</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QPushButton" name="ButtonSelectFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Tracking in a single file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButDelFile">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFilePath">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>471</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_Experimental_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>158</y>
      <width>741</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="ButtonResultFolder">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Empy results folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDelResultFolder">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelResultFolderPath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>9</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
